src/main/kotlin/org/quiltmc/community/_Constants.kt

```diff
@@ -9,7 +9,8 @@ import com.kotlindiscord.kord.extensions.utils.env
 import com.kotlindiscord.kord.extensions.utils.envOrNull
 import dev.kord.common.entity.Snowflake
 
-internal val TOKEN = env("TOKEN")
+internal val DISCORD_TOKEN = env("TOKEN")
+internal val GITHUB_TOKEN = env("GITHUB_TOKEN")
```

Comment:
This could probably be nullable, such that the extension isn't loaded if it isn't provided. You can use `envOrNull` for that.